modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/saml/SAMLECPSSOTestCase.java

```diff
@@ -174,19 +173,16 @@ private String extractDataFromResponse(HttpResponse response) throws IOException
     private void changeISConfiguration() throws Exception {
         log.info("Replacing identity.xml disabling the consent Management");
```

Comment:
Log message needs to be fixed

Reply:
Fixed - https://github.com/wso2/product-is/pull/7059